src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -43,6 +43,7 @@ const walletConnectCreated = createEvent<CreateParams<WalletConnectAccount>>();
 const walletSelected = createEvent<ID>();
 const multisigAccountUpdated = createEvent<MultisigUpdateParams>();
 const walletRemoved = createEvent<ID>();
+const walletRenamed = createEvent<Wallet>();
```

Comment:
I was thinking about this for a while and came to conclusion
_Wallet rename_ is feature that must start and end in it's own place (feature)
Otherwise we start rename inside a feature but the real action ends in ... `entity/wallet-model` that's wrong
We have kind of same approach for wallet creation, but yesterday I've moved `vaultCreated` into `ManageVault` model, it's the right place where process starts and ends
Overall each wallet creation must start and end in appropriate onboarding flow (work to be done)

So the change to be made is to move `walletRenamed` inside `RenameWallet` feature and update `walletModel.$wallet` in the end